src/software/thunderscope/thunderscope_main.py

```diff
@@ -93,6 +94,12 @@
         help="Replay folder for the yellow full_system",
         default=None,
     )
+    parser.add_argument(
+        "--verbose",
```

Comment:
semantically verbose is something that is expected to be default false. Could we instead use "--suppress_log" that is default False?

Reply:
good catch, I intended the opposite. I intended default false, and adding the --verbose flag would switch to true

Reply:
the gamecontroller + autoref flags prints are excessive